src/main/features/claim/form/models/eligibility/eligibility.ts

```diff
@@ -33,20 +36,23 @@ export class Eligibility {
   @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.HELP_WITH_FEES] })
   helpWithFees?: YesNoOption
 
-  @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.CLAIM_IS_FOR_TENANCY_DEPOSIT] })
+  @IsIn(YesNoOption.all(), {
+    message: ValidationErrors.YES_NO_REQUIRED,
+    groups: [ValidationGroups.CLAIM_IS_FOR_TENANCY_DEPOSIT]
+  })
   claimIsForTenancyDeposit?: YesNoOption
 
-  constructor (
-    claimantAddress?: YesNoOption,
-    defendantAddress?: YesNoOption,
-    claimValue?: ClaimValue,
-    singleClaimant?: YesNoOption,
-    singleDefendant?: YesNoOption,
-    eighteenOrOver?: YesNoOption,
-    governmentDepartment?: YesNoOption,
-    helpWithFees?: YesNoOption,
-    claimIsForTenancyDeposit?: YesNoOption
-  ) {
+  constructor (claimOnBehalf?: YesNoOption,
+               claimantAddress?: YesNoOption,
```

Comment:
undo this formatting change please, the formatting was better before

Reply:
As discussed: this is intelliJ's auto formatting

Reply:
@jacmurphy How is this relevant? IntelliJ auto-formatting rules are not gospel. If your IDE is making random changes throughout the code, then configure it not to.

Reply:
@radoslawn-kainos As discussed with @timja-kainos, it was auto formatted and he didn't mind which one was picked in the end.